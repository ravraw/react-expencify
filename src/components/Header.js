import React from "react";
import { NavLink } from "react-router-dom";
const Header = () => (
  <header>
    <h1>Expensify</h1>
    <NavLink to="/" activeClassName="active" exact>
      HOME
    </NavLink>
    <NavLink to="/add" activeClassName="active">
      ADD EXPENCE
    </NavLink>

    <NavLink to="/help" activeClassName="active">
      HELP
    </NavLink>
  </header>
);

export default Header;
