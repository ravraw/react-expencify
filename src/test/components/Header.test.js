import Header from "../../components/Header";
import React from "react";

import { shallow } from "enzyme";

test("Should render header correctly", () => {
  const wrapper = shallow(<Header />);
  expect(wrapper).toMatchSnapshot();
  expect(wrapper.find("h1").length).toBe(1);
});
