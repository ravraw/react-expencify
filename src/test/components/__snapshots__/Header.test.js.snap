// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="active"
    aria-current="page"
    exact={true}
    to="/"
  >
    HOME
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/add"
  >
    ADD EXPENCE
  </NavLink>
  <NavLink
    activeClassName="active"
    aria-current="page"
    to="/help"
  >
    HELP
  </NavLink>
</header>
`;
